src/pages/MyKiva/MyKivaPage.vue

```diff
@@ -211,5 +212,28 @@ export default {
 			logReadQueryError(e, 'MyKivaPage myKivaPrefetch');
 		}
 	},
+	mounted() {
+		try {
+			this.apollo.watchQuery({
+				query: gql`
+					query UserPreferences {
```

Comment:
The `myKiva.graphl` query already loads user preferences, what use case is not covered?